openstudiocore/src/model/OtherEquipment.cpp

```diff
@@ -315,6 +316,14 @@ OtherEquipment::OtherEquipment(const OtherEquipmentDefinition& definition)
   : SpaceLoadInstance(OtherEquipment::iddObjectType(),definition)
 {
   OS_ASSERT(getImpl<detail::OtherEquipment_Impl>());
+
+  /*
+   *Schedule sch = this->model().alwaysOnDiscreteSchedule();
```

Comment:
@jmarrec I think this would have been too much to default in a ctor

Reply:
this would be covered by a default schedule set so don't want to initialize a schedule